packages/terra-form-fieldset/docs/DEPENDENCIES.md

```diff
@@ -0,0 +1,20 @@
+# Dependency Information
```

Comment:
We don't need to check this in

Reply:
@are you sure I see these in packages in master branch

1) https://github.com/cerner/terra-core/blob/master/packages/terra-badge/docs/DEPENDENCIES.md

2) https://github.com/cerner/terra-core/blob/master/packages/terra-alert/docs/DEPENDENCIES.md

Reply:
This file is generated in the [postpublish script](https://github.com/cerner/terra-core/blob/master/package.json#L65) after we do a release. It doesn't hurt to check it in now, I think @emilyrohrbough is suggesting you don't need to check it in now as it will be auto-generated and checked in after the initial release.